### ISSUE TITLE
Emojify model has overfitting (Low accuracy on validaiton set)

### ISSUE THREAD

**Issue by codenigma1** [OpenSourceHelpCommunity/Questions#178]
Hi,
I am working on an independent project to predict emojis of tweets. I got the raw tweets dataset from Kaggle and did some pre-processing before use in my model. I have around 280000 tweets and seven labels ❤️😂😩🙏😭😍🔥. I trained the model on the [Kaggle notebook](https://www.kaggle.com/code/codenigma/emojify-testing/edit) and obtained **85% training accuracy** in 20 epochs but _**on the validation set accuracy is stuck at 55%**_.  I tried many time changes in architecture, and hyperparameters but did not obtain the desired result.  It would be great if any help to improve this model and look forward to discussing it with you guys.  
![c1b3448927dfc221d141b8626ad546a25fa8df9f](https://user-images.githubusercontent.com/37497128/203059330-978f7480-d714-4c6c-b13d-fd4899248554.png)

Thanks!
